README.md

```diff
@@ -711,3 +713,33 @@ cast send <SOURCE_MINTER_ADDRESS> --rpc-url avalancheFuji --private-key=$PRIVATE
 ```
 
 depending on whether you filled the [`SourceMinter.sol`](./src/cross-chain-nft-minter/SourceMinter.sol) contract with `Native (0)` or `LINK (1)` in step number 3.
+
+## CCT Standard
+
+The [Cross-Chain Token (CCT)](https://docs.chain.link/ccip/concepts/cross-chain-token/overview) standard offers a streamlined and decentralized approach to enabling token transfers across blockchains using Chainlink's Cross-Chain Interoperability Protocol (CCIP).
+
+Using CCT one can add support for cross-chain token transfers via CCIP for its own ERC20 token.
+
+This repository includes examples of ERC-20 tokens with:
```

Comment:
"This repository includes examples of ERC-20 tokens with:"

--->

"This repository includes examples of CCT-enabled ERC-20 tokens with implementations of:"